### PR TITLE
Docker setup

### LINKED ISSUE #9: docker setup for demo
setup a docker demo with
* polkadexTEE-node
* polkadexTEE-worker
* OpenFinex
* a client (demo bash script)

### PR DESCRIPTION
**Pull request by jaxter03**
**Completed**
Created Docker files for both Polkadex Node and Worker. 

**Pending**
Create Docker compose file to run in the following order:-
- Polkadex Node
- Openfinex
- Polkadex-worker
- Client

Will close #9

### CODE REVIEW

File: demo-setup/start.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Script assumes that host system has go-lang installed
+# The script assumes that openfinex opens it's port on 8001
```
Comment (haerdib) [inline demo-setup/start.sh:2]: This sripts assumes to have the openfinex components somewhere.
So I'd suggest to either:
1) Add intsructions where to get these files from and/or where to place them manually
2) add a script option somewhere

**Review by haerdib — CHANGES_REQUESTED** [review]

File: demo-setup/start.sh (hunk 1 above)
Comment (haerdib) [reply demo-setup/start.sh:2]: could you also add that the `compose` folder from openfinex should also be placed here?

Comment (Gauthamastro) [reply demo-setup/start.sh:2]: I have added the compose folder, as we discussed the openfinex can be shared internally

File: demo-setup/start.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Script assumes that host system has go-lang installed and polkadex.tar is present in the same directory and config.yaml
```
Comment (haerdib) [inline demo-setup/start.sh:1]: > and polkadex.tar is present in the same directory and config.yaml

What do you mean with the config.yaml?

Comment (haerdib) [reply demo-setup/start.sh:1]: And if we're at it: could you also add the minimum required docker compose version? 1.25 was not new enough.

Comment (Gauthamastro) [reply demo-setup/start.sh:1]: THE config folder ;)

I have added the docker-compose requirements the readme

**Comment by Gauthamastro** [comment]
@haerdib, the docker setup will not work due to how openfinex is configured and how the enclave and node interact.  @zktony  can explain to you the issues with this method. I suggest we move on with bare-metal deployment and testing everything there.

**Comment by Gauthamastro** [comment]
Closing this PR for the time being

**Pull request closed without merging**